store-api-server/src/payment/service/payment.service.ts

```diff
@@ -972,21 +975,41 @@ WHERE provider = 'simplex'
 
       events.push(event);
       for (const event1 of events) {
+        await this.createSimplexEventToDB(event1, paymentId);
         let deleteResponse = await deleteSimplexEvents(event1.event_id);
+
         if (deleteResponse?.status == 'OK') {
-          Logger.log(
-            `simplex payment DELETE event succeeded. eventId=${event1.event_id} paymentId=${paymentId}`,
-          );
+          Logger.log(`simplex payment DELETE event succeeded. eventId=${event1.event_id} paymentId=${paymentId}`);
         } else {
-          Logger.warn(
-            `simplex payment DELETE event failed. eventId=${event1.event_id} paymentId=${paymentId}`,
-          );
+          Logger.warn(`simplex payment DELETE event failed. eventId=${event1.event_id} paymentId=${paymentId}`);
         }
       }
     }
     Logger.log('getSimplexEvents FINISH successfully');
   }
 
+  private async createSimplexEventToDB(event1: any, paymentId: any) {
+    try {
+      const simplexEventCreatedResponse = await this.conn.query(
+        `
+INSERT INTO simplex_payment_event (
+  event_id, payment_id ,simplex_event_status, simplex_payment_status, 
+  partner_id, partner_end_user_id, crypto_currency, fiat_total_amount, 
+  fiat_total_amount_currency, crypto_total_amount, crypto_total_amount_currency, 
+  payment_created_at, payment_updated_at, event_timestamp
+)
+VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14 )`,
+        [event1.event_id, event1.payment.id, event1.name, event1.payment.status, event1.payment.partner_id,
+          event1.payment.partner_end_user_id, event1.payment.crypto_currency, event1.payment.fiat_total_amount?.amount,
+          event1.payment.fiat_total_amount?.currency, event1.payment.currency_total_amount?.amount, event1.payment.currency_total_amount?.currency,
+          event1.payment.created_at, event1.payment.updated_at, event1.timestamp],
+      );
+      Logger.log(simplexEventCreatedResponse.toLocaleString());
```

Comment:
I assume this is a log entry left from debugging? Don't think we should log this in deployments.

Reply:
yes, it was debugging.